core-services/image-mirroring/openshift/mapping_origin_4_11

```diff
@@ -43,6 +43,7 @@ registry.ci.openshift.org/origin/4.11:cloud-credential-operator quay.io/openshif
 registry.ci.openshift.org/origin/4.11:cloud-event-proxy quay.io/openshift/origin-cloud-event-proxy:4.11 quay.io/openshift/origin-cloud-event-proxy:4.11.0 quay.io/openshift/origin-cloud-event-proxy:latest
 registry.ci.openshift.org/origin/4.11:cloud-network-config-controller quay.io/openshift/origin-cloud-network-config-controller:4.11 quay.io/openshift/origin-cloud-network-config-controller:4.11.0 quay.io/openshift/origin-cloud-network-config-controller:latest
 registry.ci.openshift.org/origin/4.11:cluster-api-actuator-pkg quay.io/openshift/origin-cluster-api-actuator-pkg:4.11 quay.io/openshift/origin-cluster-api-actuator-pkg:4.11.0 quay.io/openshift/origin-cluster-api-actuator-pkg:latest
+registry.ci.openshift.org/origin/4.11:cluster-api-operator quay.io/openshift/origin-cluster-api-operator:4.11 quay.io/openshift/origin-cluster-api-operator:4.11.0 quay.io/openshift/origin-cluster-api-operator:latest
```

Comment:
based on my thinking in https://coreos.slack.com/archives/CBZHF4DHC/p1644597800613349?thread_ts=1644593799.371139&cid=CBZHF4DHC

and the guidlines i am trying to describe in https://github.com/openshift/openshift-docs/pull/41803

I think this should be `cluster-kube-cluster-api-operator` so long as it is going to be a CVO operator and/or can be considered to impact the config/behavior of the cluster it is running on (which cluster-api does/can through machine management, even if today we do not use it in that way)